scripts/dangerjs/detectNonApprovedDependencies/approvedPackages.ts

```diff
@@ -97,6 +97,7 @@ export default [
   'react-fela@10.6.1',
   'react-is@16.8.2',
   'react-is@16.9.0',
+  'react-is@16.13.1',
```

Comment:
needed to satisfy dangerjs violation. 

Note that this behaviour is **probably a bug** in the underlying implementation. 

**why?**

This PR is implementing deduplication and CI check of physically installed packages within our monorepo, which doesn't change nothing customer related (not touching any `package.json`). This check still resolved those changes as an error which is wrong.

I guess it physically traverses file system ?